### DISCUSSION TITLE
How do I make ghost-text appear even when I'm in the middle of the line?

### DISCUSSION THREAD

**Discussion by luiz00martins** [hrsh7th/nvim-cmp#1354] [Q&A]
It seems like ghost text only appears if you are currently in the middle of the line. Is there a way to show ghost text even if the cursor is currently in the middle of the line?

**Comment by fitrh** [comment]
You need to trigger the completion window via `cmp.complete()`

**Reply by luiz00martins** [reply]
The completion window is active, only the ghost-text isn't. Notice the difference when  the cursor is at the middle of the line, vs when it isn't:

![image](https://user-images.githubusercontent.com/43142209/208073769-c7acc46b-e2e2-45df-905d-88c87440db28.png)
![image](https://user-images.githubusercontent.com/43142209/208073832-b1ef546b-95a6-4a47-acd5-53199586bc11.png)

(note the `')` at the end)
